motif/compare.go

```diff
@@ -0,0 +1,48 @@
+package motif
+
+import (
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/numbers"
+	"github.com/vertgenlab/gonomics/numbers/parse"
+	"strings"
+)
+
+// AlmostEquals determines if floating-point numbers within two files are equal within a specified epsilon level.
+func ApproxEquals(alpha, beta string, epsilon float64) bool {
+	query, answer := fileio.Read(alpha), fileio.Read(beta)
+	indexes := []int{7, 8} // Assuming these are the indexes you want to compare
+
+	if len(query) != len(answer) {
+		//fmt.Errorf("Error: %s and %s do not have the same number of bed records...", alpha, beta)
+		return false
+	}
+
+	for i := 0; i < len(query); i++ {
+		queryFields := strings.Split(query[i], "\t")
+		answerFields := strings.Split(answer[i], "\t")
+
+		// Check if both lines have the same number of fields
+		if len(queryFields) != len(answerFields) {
+			// fmt.Errorf("Error: Line %d of files %s and %s have different number of fields", i, alpha, beta)
```

Comment:
Commented out print statements due to adding test cases that return false to improve test coverage